src/pages/en/guides/publish-to-npm.md

```diff
@@ -97,9 +100,9 @@ The module format used by Node.js and Astro to interpret your `index.js` files.
 
 We recommend using `"type": "module"` so that your `index.js` can be used as an entrypoint with `import` and `export`.
 
-#### `package.json#exports`
+#### `exports`
 
-The entry points allowed by Astro to import your component or any of its [files](#packagejsonfiles).
+The entry points allowed by Astro to import your component or any of its files.
```

Comment:
Slight tweak as `exports` is allowing Astro access rather Astro allowing the entry points access.

```suggestion
The entry points Astro is allowed to import your component or files from.
```

Reply:
I've updated via the NodeJS docs:  

The "exports" field allows defining the entry points of a package when imported by name.

This seems a level of detail comparable to what we have for the other fields. So, now we're deciding whether we like this.  :)